src/main/java/com/redhat/labs/omp/service/VersionService.java

```diff
@@ -40,8 +40,8 @@ public Version getGitApiVersion() {
         }
         version.setApplication("omp-git-api-container");
         
-        if(version.getGitTag().startsWith("v")) {
-            version.setVersion(version.getGitTag());
+        if(version.getGitTag().equals("master") || version.getGitTag().equals("latest")) {
+            version.setVersion(version.getGitTag() + "-" + version.getGitTag());
```

Comment:
Just clarification.  Is this supposed to be version.getGitTag() + version.getGitCommit()?  I may just not have the context right here.

Reply:
Definitely missed that... good catch

Reply:
yeah. basically - if you use master or latest then you are not on a release. So it'll just say

master-asdf9a8dfk3 in that case. It replaces checking if v*. Essentially means this is deployed in dev or test.

Reply:
👍